lib/dialects/postgres/query-generator.js

```diff
@@ -529,11 +529,40 @@ module.exports = (function() {
           template += ' ON UPDATE <%= onUpdateAction %>';
           replacements.onUpdateAction = attribute.onUpdate.toUpperCase();
         }
+
+        if (attribute.references.deferrable) {
+          template += ' <%= deferrable %>';
+          replacements.deferrable = attribute.references.deferrable.toString(this).toUpperCase();
```

Comment:
This is not needed, right? Using toUpperCase / toLowerCase just for the style is a performance penalty that is not needed. In my opinion a ORM should always try to build the querys as fast as possible. Other people work hard to [remove] (https://github.com/nodejs/io.js/commit/4ddd6406ce1c474e1a54a5aa9b56f7554902fd10) a single toLowerCase :D